POC/Tertiary_Task/README.md

```diff
@@ -0,0 +1,44 @@
+# ML-Herbarium Tertiary Task Swin-Transformer
+
+The original repo can be found [here](https://github.com/SwinTransformer/Swin-Transformer-Object-Detection)
+This section of the repository deals with the instance segmentation task. We use the repo stored [here](/projectnb/sparkgrp/ml-herbarium-grp/ml-herbarium-data/Tertiary_Task/Swin-Transformer-Object-Detection/) on the scc server under sparkgrp
+
+The current model can be found in `./work_dirs/flowers_36_epochs_update/epoch_36.pth` and the visualized outputs are in `outputs/plants_36_epochs_segm`
```

Comment:
I don't see these directories and files anywhere.

Reply:
@trgardos all of these files would be found on the scc at `/projectnb/sparkgrp/ml-herbarium-grp/ml-herbarium-data/Tertiary_Task/Swin-Transformer-Object-Detection/`

Reply:
I see. Can you update this README and create a PR to this branch with that information? There's no way to know that from here. I can help you with that if you need it.